examples/voltage_clamp/voltage_clamp.ipynb

```diff
@@ -0,0 +1,801 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ssis = np.array([feat_dict[\"steady_state_voltage_stimend\"][0] for feat_dict in feats])\n",
+    "plt.plot(stim_v, ssis, '.')\n",
+    "plt.xlabel(\"Stimulus Voltage (mV)\")\n",
+    "plt.ylabel(\"Steady State Current (nA)\")\n",
+    "plt.title(\"Activation steady state\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also plot the current base. Since this feature is compute on the trace before the stimulus is applied, it should be constent across the traces."
```

Comment:
compute -> computed
constent -> constant

Reply:
Fixed in latest commit